docs/dev-guide.md

```diff
@@ -310,19 +244,19 @@ sqldb.query(sql.select_question, params, ...);
 
 - Detailed descriptions of the format of each table are in the [list of DB tables](https://github.com/PrairieLearn/PrairieLearn/blob/master/database/tables/).
 
-- Each table has an `id` number that is used for cross-referencing. For example, each row in the `questions` table has an `id` and other tables will refer to this as a `question_id`. The only exceptions are the `pl_courses` table that other tables refer to with `course_id` and `users` which has a `user_id`. These are both for reasons of interoperability with PrairieSchedule.
+- Each table has an `id` number that is used for cross-referencing. For example, each row in the `questions` table has an `id` and other tables will refer to this as a `question_id`. For legacy reasons, the only exceptions are the `pl_courses` table that other tables refer to with `course_id` and `users` which has a `user_id`.
```

Comment:
```suggestion
- Each table has an `id` number that is used for cross-referencing. For example, each row in the `questions` table has an `id` and other tables will refer to this as a `question_id`. For legacy reasons, the only exceptions are the `pl_courses` table that other tables refer to with `course_id`, and the `users` table which uses `user_id` as a primary key instead of `id`.
```

Reply:
Resolved slightly differently in 938b664.